WordPress/src/main/res/values/strings.xml

```diff
@@ -884,9 +884,19 @@
     <string name="stats_collapse_content_description">Collapse</string>
     <string name="stats_item_expanded">Item expanded</string>
     <string name="stats_item_collapsed">Item collapsed</string>
-    <string name="stats_posting_activity_graph">Graph of posting activity.</string>
     <string name="stats_list_item_description">%1$s: %2$s, %3$s: %4$s</string>
 
+    <string name="stats_box_type_low">low</string>
+    <string name="stats_box_type_medium">medium</string>
+    <string name="stats_box_type_high">high</string>
+    <string name="stats_box_type_very_high">very high</string>
+
+    <string name="stats_posting_activity_action">navigate all stats for this period</string>
+    <string name="stats_posting_activity_content_description">"The days with %1$s views for %2$s are : %2$s %3$s."</string>
```

Comment:
We might want to consider ending this announcement with ", tap for more". Or is it a common practice that clicking multiple times on the same view keeps going more into detail?

Reply:
This is a good observation. I will include this change. Thanks!

Reply:
Resolved 40cd081